File: prosecurity.mobile/src/model/auth/account-info.ts

export class AccountInfo {
  public id: number;
  public username: string;
  public fio:string;

  public static create(a: any): AccountInfo {
    const ret = new AccountInfo();
    ret.assign(a);
    return ret;
  }

  private assign(a: any) {
    this.id = a.id;
    this.username = a.username;
    this.fio = a.fio;

  }
}

File: prosecurity.controller/src/kz/diploma/prosecurity/controller/model/AccountInfo.java
package kz.diploma.prosecurity.controller.model;

public class AccountInfo {
  public Long id;
  public String fio;
  public String img;
  public String username;

  public UserInfo toUserInfo() {
    UserInfo ret = new UserInfo();
    ret.displayName = fio == null ? username : fio;
    ret.username = username;
    return ret;
  }
}
